### DISCUSSION TITLE
Help! Error network

### DISCUSSION THREAD

**Discussion by Iberid1974** [amidaware/tacticalrmm#2031] [General]
![2024-10-16 15_42_21-Tactical RMM — Mozilla Firefox](https://github.com/user-attachments/assets/e019ea08-65ba-47f4-886f-7777f8acc231)
Login error
Solution? 
wget -N https://raw.githubusercontent.com/amidaware/tacticalrmm/develop/troubleshoot_server.sh
chmod +x troubleshoot_server.sh
./troubleshoot_server.sh
No, error 
![2024-10-16 15_45_11-mRemoteNG - confCons xml - vps rmm](https://github.com/user-attachments/assets/d1e665e9-3226-4731-a4e9-5418f627538d)
thanks

**Comment by Iberid1974** [comment]
add 
![2024-10-16 16_13_26-](https://github.com/user-attachments/assets/d50e9e52-a3a6-4e46-9c99-5f4ea05c3655)

**Comment by dinger1986** [comment]
What's the 200.58 IP address? Where's that come from?

**Reply by Iberid1974** [reply]
![2024-10-16 16_23_47-mRemoteNG - confCons xml - vps rmm](https://github.com/user-attachments/assets/4ab00819-28cf-4011-a363-33015597acf0)

**Reply by dinger1986** [reply]
But it's 200.58.117.54 or something. 

Whats do you get if you do `cat /etc/hosts`

**Reply by Iberid1974** [reply]
![2024-10-16 16_29_51-mRemoteNG - confCons xml - vps rmm](https://github.com/user-attachments/assets/ff8ad5a8-e272-4816-bf28-01420da86ab2)

**Reply by dinger1986** [reply]
https://docs.tacticalrmm.com/update_server/#keeping-your-lets-encrypt-ssl-certificate-up-to-date

**Reply by dinger1986** [reply]
Did you install using --insecure?

**Reply by wh1te909** [reply]
![A0oUjMC7ti](https://github.com/user-attachments/assets/bd9b3cbb-9e4e-437b-8247-583e3c6857be)

You need to open all 3 subdomains in your browser and accept the warning, see screenshot above.

**Reply by Iberid1974** [reply]
![2024-10-16 16_49_23-Tactical RMM — Mozilla Firefox](https://github.com/user-attachments/assets/db70c30b-3eac-407d-8c8d-a6ef3ddead08)
GRacias ! luego de varios intentos logre hacer que funcionara, ahora a pulir detalles
